install/playbooks/roles/dovecot/files/access-log.sh

```diff
@@ -107,9 +108,26 @@ fi
 # Remove the new lines from the details before storing them in the database
 smallDetails=$(echo "$DETAILS" | tr '\n' ';' | sed -r 's/(^;|;$)//g')
 
+# Get the Internet Service Provider from http://ip-api.com/
+# The limit is 150 requests per minute from an IP address.
+# This is particularly useful if you are using only ISP,
+# regardless of the IP address
+isp='unknown'
+
+if [ "$isPrivate" = "1"]; then
+    isp='private'
+elif [ "$ALLOW_EXTERNAL_QUERIES" = "YES" ]; then
+    data="[{\"query\": \"$IP\"}]"
+    isp=$(curl -s 'http://ip-api.com/batch?fields=isp' --data "$data" | jq '.[0].isp')
```

Comment:
I will double check this one, but you are right, it make sense.

Reply:
For this one, it is the other way around ! Since the value is going to be used inside an SQL query, it should be escaped if it contains single quotes.
Also, I discovered there is a "line" export for ip-info, which remove the need to use jq, and simplifies the query. See the last version.
Added a curl timeout, and fail silently. Therefore, the final test change from "null" to an empty string.